src/main/resources/db/changelog/1.0/2020-10-21-change-table-category.yaml

```diff
@@ -0,0 +1,14 @@
+databaseChangeLog:
+  - changeSet:
+      id: SUP-10-1
+      author: alexander_popovich
+      changes:
+        - addColumn:
+            tableName: category
+            columns:
+              - column:
+                  name: is_active
```

Comment:
its not a good idea to place prefixes such 'is' in database column names, as there is a possible misunderstanding in the naming of fields in java (if use default mapping strategy - which is preferable)

Reply:
done